contracts/transfer/circuits/src/gadgets/balance.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) DUSK NETWORK. All rights reserved.
+// Licensed under the MPL 2.0 license. See LICENSE file in the project root for details. 
+
+use dusk_plonk::constraint_system::ecc::scalar_mul::fixed_base::scalar_mul;
+use dusk_plonk::jubjub::{
+    Fr, AffinePoint, GENERATOR_EXTENDED, GENERATOR_NUMS_EXTENDED,
+};
+use dusk_bls12_381::Scalar;
```

Comment:
Not needed, prelude is indeed exporting BlsScalar and JubJubScalar types.

Reply:
If I remove JubJub & Bls and keep prelude. Then it fails and does not compile. With a message 'not found in this scope'.

Reply:
I'm not saying that you should remove the other inputs. But `Fr` and `Scalar` yes.